### DISCUSSION TITLE
双显示器无法正常启动进入系统

### DISCUSSION THREAD

**Discussion by coopsrc** [hackintosh-club/intel-nuc10#65] [General]
OC 0.7.8
Big Sur 11.6.5

双显示器环境
主显示器：4K，type-c连接
副显示器：1080P，HDMI连接

同时连接两个显示器启动时，可以进入读条阶段，读条之后显示器显示无信号，电脑似乎未关机（电源键在亮）。
断开副显示器，强制重新启动，正常。连接副显示器，正常使用。

**Comment by jlzhou** [comment]
try to set BIOS: Advanced - IGD Primary Video Port as HDMI, IGD Secondary Port as None.

Please reply result here if you get a try

**Comment by coopsrc** [comment]
Both options work

IGD Primary Video Port as thunderbolt, IGD Secondary Port as None
IGD Primary Video Port as HDMI, IGD Secondary Port as  None

**Reply by kclgh** [reply]
Thank You! yours and jizhou's answers solved my same problems!
